EasyPost/Constants.cs

```diff
@@ -90,5 +90,17 @@ public static class ErrorMessages
             internal const string ApiErrorDetailsParsingError = "RESPONSE.PARSE_ERROR"; // not for public consumption
             public const string CouldNotPassClient = "Could not pass client to {0}.";
         }
+
+        public static class CarrierAccountTypes
+        {
+            public const string FedExAccountType = "FedexAccount";
+            public const string UpsAccountType = "UpsAccount";
+
+            internal static List<string> CustomWorkflowCarrierAccountTypes => new List<string>
+            {
+                FedExAccountType,
+                UpsAccountType
```

Comment:
Nit: No need to define these variables, I would just pass them directly

Reply:
This way, users can use the constants if they'd like

Reply:
When building their data for the API call. That way they don't have to remember if it's "FedExAccount" or "FedexAccount" (it's the latter)
<img width="657" alt="image" src="https://user-images.githubusercontent.com/17054780/203618889-1a8661b4-26bd-461f-9527-52c6d521184b.png">

Reply:
Ah, ok! Makes sense to me now